core/orchestrator.go

```diff
@@ -753,13 +753,21 @@ func (rtm *RemoteTranscoderManager) Manage(stream net.Transcoder_RegisterTransco
 	rtm.liveTranscoders[transcoder.stream] = transcoder
 	rtm.remoteTranscoders = append(rtm.remoteTranscoders, transcoder)
 	sort.Sort(byLoadFactor(rtm.remoteTranscoders))
+	if monitor.Enabled {
+		monitor.SetTranscodersNumber(int64(len(rtm.liveTranscoders)))
+		monitor.SetTranscodersCapacity(int64(rtm.totalCapacity()))
+	}
 	rtm.RTmutex.Unlock()
 
 	<-transcoder.eof
 	glog.Infof("Got transcoder=%s eof, removing from live transcoders map", from)
 
 	rtm.RTmutex.Lock()
 	delete(rtm.liveTranscoders, transcoder.stream)
+	if monitor.Enabled {
+		monitor.SetTranscodersNumber(int64(len(rtm.liveTranscoders)))
+		monitor.SetTranscodersCapacity(int64(rtm.totalCapacity()))
```

Comment:
Would it also be useful to expose the current load for the transcoder? Or even the load as a percentage of capacity? That seems like good information to track to get a feel for the overall work distribution on the O.

Reply:
Good idea. Exposed 'load' metric - just total value of transcoders `load` field. Percentage can be easily calculated in Prometheus itself.